src/concrete/ml/onnx/convert.py

```diff
@@ -167,6 +168,10 @@ def get_equivalent_numpy_forward_from_onnx(
             forward.
         check_model (bool): set to True to run the onnx checker on the model.
             Defaults to True.
+        lsbs_to_remove (List[int]): Contains the values of the least significant bits to
+            remove during tree traversal. The first value pertains to the first comparison (either
```

Comment:
I still don't understand why we have something very specific to trees in a very general function like this one ! 

once we have the truncated feature, will this be removed ? : 
- if so : that means we'll need to break the API when integrating the truncate feature (and thus bump CML to 2.0)
- it not : is it really the only way to do so ? If we truly don't have other solutions, I would add quite a few things in the following : 
    - insist more on the fact that this feature is only used for trees 
    - maybe rename it to something like `lsbs_to_remove_for_trees`
    - add an assert that makes sure the model is representing a tree based model (not sure how to do that with the onnx though

Reply:
- It won't be removed with truncate, it will just work better. I don't consider it to be breaking API as this is not a public function
- I agree with the second point. The assert could be done by checking there is a Matmul -> comparison -> matmul -> equality structure.

Still, this is not a function a user would call so I don't think we need a complicated test to check if it's a tree, something simple will do.

Reply:
Or we could just put the lsbs_to_remove in the onnx thus allowing us to keep these functions clean... I still don't understand why we don't want that.

Reply:
When we integrate 'truncate,' everything will disappear!
  -  'Truncate' will replace 'rounding_bit_pattern' in ops_impl.py.
  -  We will add an auto-truncate feature in the tree class that will be adjusted during the 'fit' call to compute the LSBs to remove.
   - This auto-truncate will be passed through all these functions